src/wizard/interfaces.ts

```diff
@@ -103,6 +103,13 @@ export interface WizardProps extends BaseComponentProps {
    */
   allowSkipTo?: boolean;
 
+  /**
+   * Specifies right-aligned custom primary actions for the wizard. Overwrites existing buttons (e.g. Cancel, Next, Finish, ...).
```

Comment:
This `...).` punctuation in the end looks strange, you will need to check with a designer if there is a better way